src/node_worker.cc

```diff
@@ -41,6 +42,19 @@ using v8::Value;
 namespace node {
 namespace worker {
 
+namespace {
+// The worker count is intentionally process scoped so
+// we can track the total number of active workers
+// across the process.
+int64_t active_worker_count = 0;
+
+// The active worker warning will be emitted each time
+// the count exceeds the threshold but only once each
+// time it is exceeded.
+bool active_worker_count_warning = true;
```

Comment:
These should be atomic and set/checked atomically

Reply:
heh.. doh...